splunk_add_on_ucc_framework/install_python_libraries.py

```diff
@@ -69,38 +95,39 @@ def install_python_libraries(
     path_to_requirements_file = os.path.join(source_path, "lib", "requirements.txt")
     if os.path.isfile(path_to_requirements_file):
         logger.info(f"Installing requirements from {path_to_requirements_file}")
-        if includes_ui:
-            ucc_library_present = _check_ucc_library_in_requirements_file(
-                path_to_requirements_file
-            )
-            if not ucc_library_present:
-                raise SplunktaucclibNotFound(
-                    f"splunktaucclib is not found in {path_to_requirements_file}. "
-                    f"Please add it there because this add-on has UI."
-                )
         if not os.path.exists(ucc_lib_target):
             os.makedirs(ucc_lib_target)
         install_libraries(
-            path_to_requirements_file,
-            ucc_lib_target,
-            python_binary_name,
+            requirements_file_path=path_to_requirements_file,
+            installation_path=ucc_lib_target,
+            installer=python_binary_name,
         )
-
-        if os_libraries:
-            install_os_dependent_libraries(
-                ucc_lib_target, python_binary_name, os_libraries
+        if includes_ui and not _pip_is_lib_installed(
+            installer=python_binary_name, libname="splunktaucclib"
```

Comment:
Why can't we install it when missing to the latest one compatible with latest UCC?

Reply:
We can, but I'd rather let user decide what exactly they want to install. It can interfere with other libraries add-on developer wants to install.